### DISCUSSION TITLE
Installation instructions

### DISCUSSION THREAD

**Discussion by JoLukas** [macedonga/beunblurred#70] [Q&A] [answered]
Hey, I'm not really good with the terminal in MacOS. I've already tried to generate more detailed instructions with ChatGPT, but unfortunately in vain. Can someone perhaps provide more detailed instructions for local installation for us "beginners" ;-) ? I hope someone can do this.

**Comment by GWPvGWP** [comment, marked as answer]
Video guides are now available and can be found in the 'Readme'
